src/zentables/__init__.py

```diff
@@ -13,6 +13,6 @@
     df.zen.pretty()
 """
 
-from .zentables import ZenTablesAccessor, set_options, _do_suppression
+from .zentables import ZenTablesAccessor, _do_suppression, set_options
```

Comment:
I would still remove `_do_suppression` here. See my other comments on making this work in test scripts